test/functional/page_objects/visualize_chart_page.ts

```diff
@@ -45,32 +47,32 @@ export function VisualizeChartPageProvider({ getService, getPageObjects }: FtrPr
     /**
      * Is new charts library enabled and an area, line or histogram chart exists
      */
-    private async isVisTypeXYChart(): Promise<boolean> {
+    public async isNewLibraryChart(chartSelector: string): Promise<boolean> {
```

Comment:
update the comment to include also the pie charts
side note: having a single setting for the legacy vislib means that when a user discovers a bug on the pie or xys solvable switching back to the legacy implementation, it causes going back also on every charts: both pie and xys, causing a major change in every chart. Probably is worth chatting about having different settings for each legacy feature to minimize changes when switch back

Reply:
🔝 this actually is also something that I like to discuss before merging

Reply:
Yes Marco this is correct. We have already discussed it with @timroes and we decided to go with one setting for now.